cmd/apiserver/app/server/run_server.go

```diff
@@ -35,7 +35,8 @@ func RunServer(opts *ServiceCatalogServerOptions, stopCh <-chan struct{}) error
 	if stopCh == nil {
 		/* the caller of RunServer should generate the stop channel
 		if there is a need to stop the API server */
-		stopCh = make(chan struct{})
+		// stopCh = server.SetupSignalHandler()
+		panic("stop channel was not set when starting the api server")
```

Comment:
also, i feel like this should be tested

Reply:
Yes, i've change the code. when RespectsStopCh=true, stopCh is nil,, ,error return, so os.Exit(1) will be called at hyperkube.Run. also  i add the tests